daml-lf/encoder/BUILD.bazel

```diff
@@ -10,19 +10,14 @@ load(
 )
 load(
     "//daml-lf/language:daml-lf.bzl",
+    "LF_VERSIONS",
     "lf_dev_version",
     "lf_latest_version",
     "lf_preview_version",
     "lf_stable_version",
 )
 
-lf_targets = [
-    "1.6",
-    "1.7",
-    "1.8",
-    "1.11",
-    "1.dev",
-]
+lf_targets = ["1.dev" if ver == "dev" else ver for ver in LF_VERSIONS]
```

Comment:
Actually isn’t this just `COMPILER_LF_VERSIONS`?

Reply:
no. `COMPILER_LF_VERSIONS` still does not include "1.12".

Reply:
I can clean this up when I add the compiler support (probably by moving this to the other file and calling it `ENCODER_LF_VERSIONS` to make its relation with `COMPILER_LF_VERSIONS` more explicit).